### ISSUE TITLE
issue with libgobject

### ISSUE THREAD

**Issue by lcnature** [pupil-labs/pupil#2297]
Hello,
I installed this package in Ubuntu 20.0 following the instruction of 18.0. When I tried to run `python main.py capture`, I got the following error:
```
  File "/home/{myusername}/anaconda3/envs/pupil/lib/python3.10/site-packages/av/__init__.py", line 9, in <module>
    from av._core import time_base, pyav_version as __version__
ImportError: /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0: undefined symbol: ffi_type_uint32, version LIBFFI_BASE_7.0
```
Do you have any clue of what might have caused this?
Things I have done slightly different from the default installation:
I installed pye3d 0.3.1 by `pip install pye3d` because when I installed based on requirements.txt, it complained that it failed building wheel for pye3d. 
I also tried to use the custom built libusb.

Thanks!

**Comment by papr** [comment]
Hi!

Did you know that we offer pre-built bundled applications? You can download them [here](https://github.com/pupil-labs/pupil/releases/tag/v3.5#user-content-downloads). You should have fewer issues with it than running from source.

Should you want to run from source nonetheless, please try checking out the [`develop`](https://github.com/pupil-labs/pupil/tree/bundle-refactor) branch and installing its requirements into a fresh virtual environment.

That said, your issue does not seem to be related to this repository specifically. It rather seems like there is an issue with the system library `/usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0`. Unfortunately, I do not have any insights as to what might be causing it.

**Comment by papr** [comment]
This might help https://medium.com/@Aenon/glib-symbol-lookup-bug-in-debian-or-ubuntu-def678dab5a6

**Comment by papr** [comment]
Closing due to inactivity

**Comment by lcnature** [comment]
Thanks! Sorry that I forgot to follow up. I tried to install it from source partially because I found that somehow I could not record sound from any sound input device connected to the laptop when using Pupil Core so I was hoping that installing from source might fix it. And I was also exploring the possibility of using the code of gaze detection for dataset collected from another manufacture. But I have not got back to that problem since. So it is fine to close it for now and I will refer to what you suggest when I try again in the future.
